src/overscroll-color-context.tsx

```diff
@@ -51,15 +51,44 @@ const TopOverscrollColor = () => {
     return null;
 };
 
+const useBodyBackgroundColor = () => {
+    const [bodyBackgroundColor, setBodyBackgroundColor] = React.useState<string>('transparent');
+
```

Comment:
Why transparent? Until now we were using the body's backgroundColor as default.

Reply:
yes, and we also avoid hydration issues, we can't read body background in SSR